front/src/utils/index.ts

```diff
@@ -23,7 +23,13 @@ export function getLogoUrlFromDomainName(
 
   if (!sanitizedDomain) return;
 
-  for (const prefix of ['', 'www.', 'https://']) {
+  const prioritizeWWW = ['apple.com'].includes(sanitizedDomain);
+
+  const prefixes = prioritizeWWW
+    ? ['www.', '', 'https://']
+    : ['', 'www.', 'https://'];
+
+  for (const prefix of prefixes) {
```

Comment:
It appears that when searching for the apple favicon using a URL without the `www` prefix, it returns the `temple` favicon instead. To address this issue, we decided to switch it up and begin the search with the `www` prefix. However, we recognize that this solution is not scalable as we cannot maintain different priorities for various websites.

Reply:
Ok, we are putting in place our own favicon server to fix the issue at the root cause :)

Reply:
favicon.twenty.com/microsoft.com